Pyomero upload

jhay-epcc opened the pull request:
**Update to pyomero-upload version 5.4.10_1.2.0**

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

jhay-epcc commented:
@BiocondaBot please add label

raivivek commented on recipes/pyomero-upload/meta.yaml:
```diff
@@ -1,12 +1,13 @@
 source:
-  url: https://github.com/SynthSys/pyOmeroUpload/archive/v1.1.0.tar.gz
-  sha256: a26890aa34fec7fe007a17f2552333723b5fbf4c2529d36b433bed70da6ae7fa
+  url: 'https://github.com/SynthSys/pyOmeroUpload/archive/v5.4.10-1.2.0.tar.gz'
```
Should use {{PYOMERO_UPLOAD_VERSION}} here

jhay-epcc commented:
Hi, I have replaced the version number for the release archive and I have removed all of the pinned versions for the secondary dependencies. The other pinned versions are required. Please merge, thanks.

@BiocondaBot please add label

jhay-epcc commented:
@BiocondaBot please add label

raivivek commented:
@bioconda-bot please merge
